### ISSUE TITLE
Updating link to Snap maintainers in CONTRIBUTING.md

### ISSUE THREAD

**Issue by IzabellaRaulin** [intelsdi-x/snap-pluginsync#40]
The next plugin sync action should take into consideration fixing link to Snap maintainers in **CONTRIBUTING.md** across all plugins.

## Currently:
(point to README.md#maintainers which does NOT exist)
```
If you have questions feel free to contact the [maintainers](https://github.com/intelsdi-x/snap/blob/master/README.md#maintainers).
```

## It should be updated on:
(should point to docs/MAINTAINERS.md#snap-maintainers or just to docs/MAINTAINERS.md)
```
If you have questions feel free to contact the [maintainers](https://github.com/intelsdi-x/snap/blob/master/docs/MAINTAINERS.md#snap-maintainers).
```
or 
```
If you have questions feel free to contact the [maintainers](https://github.com/intelsdi-x/snap/blob/master/docs/MAINTAINERS.md).
```

**Comment by IzabellaRaulin** [comment]
Related to changes made by https://github.com/intelsdi-x/snap/pull/1311

**Comment by nanliu** [comment]
Fixed by #39
